### ISSUE TITLE
questions about crossattn in CustomDiffusionAttnProcessor

### ISSUE THREAD

**Issue by qiguming** [adobe-research/custom-diffusion#80]
if crossattn:
            detach = torch.ones_like(key)
            detach[:, :1, :] = detach[:, :1, :]*0.
            key = detach*key + (1-detach)*key.detach()
            value = detach*value + (1-detach)*value.detach()

Why stop the gradient of the first key-value pair here?
